lib/main.py

```diff
@@ -134,6 +139,13 @@ def main() -> None:
 
             elif args.command == 'fetch-scala':
                 fetch_scala(ws, args, agg=False)
+
+            elif args.command == 'inspect':
+                if args.dot or args.tree:
+                    inspect_tree(ws, args)
+                else:
+                    log.error("wit inspect must be run with the --tree flag")
+                    sys.exit(1)
```

Comment:
better to use the `error` method here in case we want to refactor error reporting: https://github.com/sifive/wit/blob/3c23d2217c5c23eb99a4e9b1e160fc33d9eb84b8/lib/common.py#L9

Also you should at least update this error message to mention that it can be run with `--dot` instead. Alternatively, you could look into using Argparse for this and make it know that one of these must be passed: https://stackoverflow.com/a/26992149/2483329, there are mutually exclusive groups and maybe you can make it mutually exclusive and required. Not sure if this is the best way, but worth considering